lib/OpenQA/Parser/Format/IPA.pm

```diff
@@ -0,0 +1,148 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package OpenQA::Parser::Format::IPA;
+
+# Translates to JSON IPA format -> OpenQA internal representation
+# The parser results will be a collection of OpenQA::Parser::Result::IPA::Test
+use Mojo::Base 'OpenQA::Parser::Format::JUnit';
```

Comment:
Why JUnit and not XUnit?

Reply:
Because it has already the attribute declaration of include_result, and don't need nothing from XUnit ( we actually could move it to the Base format )

Reply:
Moved to base format so it's more clear